packages/axios-extension/src/abap/abap-service-provider.ts

```diff
@@ -1,21 +1,20 @@
 import { ServiceProvider } from '../base/service-provider';
 import type { CatalogService } from './catalog';
 import { V2CatalogService, V4CatalogService } from './catalog';
-import type { AtoSettings } from './adt-catalog';
 import { Ui5AbapRepositoryService } from './ui5-abap-repository-service';
 import { AppIndexService } from './app-index-service';
 import { ODataVersion } from '../base/odata-service';
 import { LayeredRepositoryService } from './lrep-service';
-import { AdtServiceName, AdtServiceConfigs, parseAtoResponse, TenantType } from './adt-catalog';
-import type { AbapServiceProviderExtension } from './abap-service-provider-extension';
-import { getTransportRequestList } from './adt-catalog/handlers/transport';
 import { AdtCatalogService } from './adt-catalog/adt-catalog-service';
-import type { AdtCollection, TransportRequest } from './types';
+import type { AtoSettings } from './types';
+import { TenantType } from './types';
+// eslint-disable-next-line @typescript-eslint/consistent-type-imports
```

Comment:
Add a comment as to why do we need this?

Reply:
Added comment

Reply:
Something along the lines of?
```suggestion
// Can't use an `import type` here. We need the classname at runtime 
// to create object instances
```

Reply:
This is better. Adopted.